docs/api/profiles.md

```diff
@@ -0,0 +1,86 @@
+# Profiles
+
+Profiles contain information about a User. A User does not need a Profile, but a Profile is always created
+together with a User.
```

Comment:
If a profile is linked to a user with an existing profile, will it be overwritten or an error occurs?